nexus/db-queries/src/db/datastore/zpool.rs

```diff
@@ -310,4 +311,27 @@ impl DataStore {
 
         Ok(SledUuid::from_untyped_uuid(id))
     }
+
+    pub async fn zpool_set_control_plane_storage_buffer(
+        &self,
```

Comment:
Do we log when this value is changed?  If not, could we log it somewhere?

Reply:
Definitely: 264465789